lib/usdUfe/python/wrapEditRouter.cpp

```diff
@@ -77,9 +77,34 @@ class PyEditRouter : public UsdUfe::EditRouter
             TF_WARN("%s", ex.what());
             throw;
         }
-        boost::python::extract<PXR_NS::VtDictionary> extractedDict(dictObject);
-        if (extractedDict.check()) {
-            routingData = extractedDict;
+
+        const boost::python::object items = dictObject.items();
```

Comment:
Could you add a comment in the code explaining what the new code is doing better than the old one? I'm sure you added all this code to fix an issue, I'd just like to understand what was going wrong and future reader will appreciate knowing too.

Reply:
Oh, it is the bit about allowing to pass an edit target object instead of just a layer! Good work, but do document it in the code.